### PR TITLE
ci: ignore CVE-2024-0567 and CVE-2023-5981 to fix ci

### PR DESCRIPTION
**Pull request by salaxander**
Describe the changes in this pull request using active verbs such as _Add_, _Remove_, _Replace_ ...

Closes #<_issue_ID_>

### CODE REVIEW

File: integration/fixtures/trivy_ignore.rego [hunk 1]
```diff
@@ -10,7 +10,9 @@ ignore_vulnerability_ids := {
     # bind-license package version "9.11.4-26.P2.el7_9.15" does not exist
     "CVE-2023-3341",
     # libssh2 package version "1.8.0-4.el7_9.1" does not exist yet
-    "CVE-2020-22218"
+    "CVE-2020-22218",
```
Comment (sozercan) [inline integration/fixtures/trivy_ignore.rego:13]: can you check if we can clean up these now?

Comment (sozercan) [reply integration/fixtures/trivy_ignore.rego:13]: otherwise LGTM pending tests

**Comment by salaxander** [comment]
@sozercan would we still expect the CI to be failing with this PR?

**Comment by sozercan** [comment]
@salaxander sounds like issues have not been addressed. you can test individually or add them back in

**Comment by sozercan** [comment]
reverting to unblock #508

**Review by sozercan — APPROVED** [review]

**Comment by salaxander** [comment]
Still seeing failures :(. The tests orginally failed before I had cleared those other 2 CVEs too

**Comment by sozercan** [comment]
ah i think it's because of the bug #508 fixes, so chicken and a egg. i'll merge this in

**Comment by ashnamehrotra** [comment]
the trivy command and file looks correct in the test logs, https://github.com/ashnamehrotra/copacetic/actions/runs/7938093411/job/21676447774#step:9:101 but copa is still checking for the error https://github.com/ashnamehrotra/copacetic/actions/runs/7938093411/job/21676447774#step:9:1897. Maybe it could be something within trivy?

**Comment by sozercan** [comment]
looks like `CVE-2023-5981` is also marked as needing u5 so ignoring that too, this should fix it i think

**Comment by sozercan** [comment]
@ashnamehrotra it was hard to test since it was just locking without your pr, i merged them both in #509 to test

**Pull request merged**